test/CVXStrategy.js

```diff
@@ -597,9 +604,9 @@ describe("CVXStrategy", function () {
         expect(
             Number(await newStrategy.balanceOfCurveLPUnstaked())
         ).to.be.equal(Number(curveLPStaked));
-
-        await newStrategy.harvest();
-
+        console.log(await newStrategy.balanceOfCurveLPUnstaked());
```

Comment:
Could be represented in one line: `await (await ...).wait()`

Reply:
same as below